### PR TITLE
Add notes abot AWS Elasticsearch service

### PR DESCRIPTION
**Pull request by bernd**
This adds two short sentences about using the AWS Elasticsearch service.

### CODE REVIEW

**Comment by joschi** [comment]
@bernd Do you think it would make sense to add a separate section for using the AWS Elasticsearch Service with Graylog which also provides details how to set up an Elasticsearch cluster on AWS?

**Comment by bernd** [comment]
@joschi I think we can create a separate section if we need to document more about this. Setting up a ES cluster in AWS is well documented in the official AWS documentation. I don't think it makes sense to duplicate this in our documentation where we have to maintain it.

The only thing I had to do after creating the ES cluster in AWS was to copy and paste the URL into the graylog config file. (`elasticsearch_hosts`)

File: pages/configuration/elasticsearch.rst [hunk 1]
```diff
@@ -16,6 +16,8 @@ Starting with version 2.3, Graylog uses the HTTP protocol to connect to your Ela
 
 .. caution:: Graylog 2.3 **does not** work with Elasticsearch 6.x yet!
 
+.. note:: Graylog 2.3 works with the `AWS Elasticsearch service <https://aws.amazon.com/elasticsearch-service/>`_ but make sure to use version **5.3** or later.
```
Comment (joschi) [inline pages/configuration/elasticsearch.rst]: Minor nitpick: The official full name is "Amazon Elasticsearch Service", not "AWS Elasticsearch service".

Comment (joschi) [reply pages/configuration/elasticsearch.rst]: Maybe rephrase it to:
```text
Graylog works fine with the `Amazon Elasticsearch Service <https://aws.amazon.com/elasticsearch-service/>`_ using **Elasticsearch 5.3.x** or later.
```

**Review by joschi — CHANGES_REQUESTED** [review]

**Review by joschi — APPROVED** [review]
